app/assets/js/components/Work/Tabs/Preservation/FileSetModal.jsx

```diff
@@ -228,17 +251,31 @@ function WorkTabsPreservationFileSetModal({
                 </UIFormField>
 
                 {watchRole && (
-                  <div className="block mt-5">
-                    <WorkTabsPreservationFileSetDropzone
-                      currentFile={currentFile}
-                      acceptedFileTypes={acceptedFileTypes}
-                      fileSetRole={watchRole}
-                      handleRemoveFile={resetForm}
-                      handleSetFile={handleSetFile}
-                      uploadProgress={uploadProgress}
-                      workTypeId={workTypeId}
-                    />
-                  </div>
+                  <>
+                    <div css={sectionCss}>
+                      <h3 css={sectionHeaderCss}>Option 1: Drag and Drop File</h3>
```

Comment:
Will let @mathewjordan make the call, but maybe as a general pattern we lean into existing styles where possible?

Reply:
Up for whatever you and @mat decide for sure.